Ntuplizer/test/ntuplize_cfg.py

```diff
@@ -285,27 +302,26 @@
     from UWVV.AnalysisTools.templates.ZZSkim import ZZSkim
     FlowSteps.append(ZZSkim)
 
-elif zl or z:
+elif zl or z or wz:
     from UWVV.AnalysisTools.templates.ZPlusXBaseFlow import ZPlusXBaseFlow
     FlowSteps.append(ZPlusXBaseFlow)
-    if zl:
+    if wz:
```

Comment:
Some items in here need to be done for Z+l too. You many need multiple if blocks.

Reply:
I just changed it to wz or zl, WZ IDs could be turned off for zl but I don't think they hurt anything.